### PR TITLE
#168292445 Refactor file and folder structure of the codebase

### PR DESCRIPTION
**Pull request by kdaglas**
## Description ##
This PR is to change the folder structuring and file formatting of our codebase

## Type of change ##
Please select the relevant option
- [x] Bugfix(a non-breaking change which fixes an issue)
- [x] This change requires a documentation update

## Necessary documentation? ##
Screen Shot 2019-09-10 at 10 12 46 AM

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Related Pivotal Tracker stories ##
[168292445](https://www.pivotaltracker.com/story/show/168292445)

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -6,21 +6,34 @@ import { ToastrModule } from 'ngx-toastr';
...
 import { RouterModule } from '@angular/router';
-import { AuthLayoutComponent } from './layouts/auth-layout/auth-layout.component';
-import { CommonLayoutComponent } from './layouts/common-layout/common-layout.component';
-import { ComponentsModule } from './components/components.module';
+import { FeaturesComponent } from './modules/features/features.component';
+import { SharedModule } from './modules/shared/shared.module';
 import { AngularFontAwesomeModule } from 'angular-font-awesome';
-import { JwtInterceptor } from './helpers/jwt.interceptor';
-import { ErrorInterceptor } from './helpers/error.interceptor';
+import { JwtInterceptor } from './interceptors/jwt/jwt.interceptor';
```
Comment (espoirMur) [inline src/app/app.module.ts]: What do you think about using absolutes imports bro?

**Review by espoirMur — COMMENTED** [review]
Bro just a quick one, which type of import should we use?
Absolute or relative?

File: src/app/app.module.ts (hunk 1 above)
Comment (kdaglas) [reply src/app/app.module.ts]: we are supposed to use absolute paths... thanks for reminding me, thats what we are to use... let change those real quick @espoirMur

File: src/app/components/home/home.component.spec.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { HomeComponent } from './home.component';
-import { ComponentsModule } from '../../components/components.module';
+import { SharedModule } from '../../modules/shared/shared.module';
```
Comment (JamesMudidi) [inline src/app/components/home/home.component.spec.ts]: Since we are using absolute paths, wouldn't this have to change as below?.
```suggestion
import { SharedModule } from 'src/app/modules/shared/shared.module';
```

**Comment by shemaeric** [comment]
@kdaglas this is a good job, I would like to point some few concerns:

**1. I have noticed that there is a change in the navbar for the `login` and `signup` buttons.**

Screen Shot 2019-09-09 at 20 01 19

**I think these buttons would look better with the existing design that was there**

Screen Shot 2019-09-09 at 20 03 01

**2. Also, the design for the dropdown menu  on small screens has changed**

Screen Shot 2019-09-09 at 20 45 30

**The design that was there**

Screen Shot 2019-09-09 at 20 46 14

**3. I have noticed that after logging in, the `navbar` doesn't change accordingly, it waits until you refresh. means I can still see the login and sign up buttons when logged in.**

Screen Shot 2019-09-09 at 20 15 08

File: src/app/app.module.ts [hunk 3]
```diff
@@ -6,21 +6,34 @@ import { ToastrModule } from 'ngx-toastr';
...
+  InternationalPaymentStatusComponent
+} from 'src/app/modules/features/components/payment/international-payment-status/international-payment-status.component';
 import { BrowserModule } from '@angular/platform-browser';
+import { AuthenticationComponent } from './modules/authentication/authentication.component';
+import { TermsPageComponent } from './components/terms/terms.component';
+import { PropertiesComponent } from './components/properties/properties.component';
+import { NoPropertiesComponent } from './components/properties/no-properties/no-properties.component';
+import { FeaturesModule } from './modules/features/features.module';
+import { AuthenticationModule } from './modules/authentication/authentication.module';
+import { PropertyDetailsComponent } from './components/property-details/property-details.component';
```
Comment (Niyitangasam) [inline src/app/app.module.ts]: Great job @kdaglas, however, it looks like you you have forgotten to change all imports to absolutes.

**Comment by Tevinthuku** [comment]
Hey @kdaglas , nice work, Im getting this error though, right in the home page.
image

File: src/app/app.module.ts (hunk 3 above)
Comment (kdaglas) [reply src/app/app.module.ts]: am working on that they are so many sometimes i lose track, but am working on it

**Comment by kdaglas** [comment]
@Tevinthuku am looking through this issue, give me a few minutes

File: src/app/app.component.ts [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+  // public hide;
+  // constructor(private route: ActivatedRoute) {
+  //   route.url.subscribe((url) => url.map(u => {
+  //     console.log(u.path);
+  //     if (u.path === '/login') {
+  //       this.hide = true;
+  //     }
+  //     this.hide = false;
+  //   }));
+  // }
```
Comment (MBenedicte) [inline src/app/app.component.ts]: Are these comments necessary?

File: src/app/helpers/tests/spies.ts [hunk 5]
```diff
@@ -24,7 +24,8 @@ export const profileServiceSpy = createSpyObj('ProfileService', [
   'getProfile',
   'updateProfile',
   'pushProfile',
-  'getDeposits'
+  'getDeposits',
+  // 'userProfile$',
```
Comment (MBenedicte) [inline src/app/helpers/tests/spies.ts]: I think this also should be removed

File: src/app/modules/features/components/payment/payment.module.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+//   ],
+//   declarations:
+//     [TokenizedCardComponent,
+//       PinValidateComponent,
+//       PinPaymentComponent,
+//       InternationalPaymentComponent,
+//       InputErrorsComponent],
+//
+// })
+// export class PaymentModule { }
```
Comment (MBenedicte) [inline src/app/modules/features/components/payment/payment.module.ts]: What happened to this file? Is it no longer used? I think you can remove it instead of commenting on everything

File: src/app/modules/features/components/profile/financial-information/financial-information.component.spec.ts [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+//   //   expect(component.cardInfo).toBeDefined();
+//   // });
+//   //
+//   // it('Should display savings ', () => {
+//   //   fixture.detectChanges();
+//   //
+//   //   const el = fixture.debugElement.query(By.css('.current-savings h2')).nativeElement.textContent;
+//   //   expect(el).toEqual(` ${mockDepositsResponse.data.transactions[0].total_amount_paid} `);
+//   // });
+// });
```
Comment (MBenedicte) [inline src/app/modules/features/components/profile/financial-information/financial-information.component.spec.ts:108]: The same as this file

File: src/app/modules/features/components/profile/financial-information/financial-information.component.ts [hunk 8]
```diff
@@ -42,7 +43,9 @@ export class FinancialInformationComponent implements OnInit, OnDestroy {
   }
 
   getCardInfo() {
+    // console.log(this.profileService.userProfile$.subscribe())
```
Comment (MBenedicte) [inline src/app/modules/features/components/profile/financial-information/financial-information.component.ts]: Remove the comment if they are no longer used

File: src/app/modules/features/components/profile/financial-information/financial-information.component.ts [hunk 9]
```diff
@@ -42,7 +43,9 @@ export class FinancialInformationComponent implements OnInit, OnDestroy {
...
     this.profileService.userProfile$.subscribe(response => {
+      console.log(response);
```
Comment (MBenedicte) [inline src/app/modules/features/components/profile/financial-information/financial-information.component.ts:47]: the same as the console.log, remove it.

File: src/app/modules/shared/components/navbar/navbar.component.html [hunk 10]
```diff
@@ -56,16 +56,23 @@
...
+          *ngIf="!authenticated"
+          class="btn-auth"
+          routerLink="/register"
+          routerLinkActive="active"
+        >
+          Sign up
+        </button>
+<!--        <button class="btn" type="submit">-->
+<!--          Log in-->
+<!--        </button>-->
```
Comment (MBenedicte) [inline src/app/modules/shared/components/navbar/navbar.component.html]: Remember to remove the comments, if they no longer used

File: src/app/modules/shared/components/navbar/navbar.component.scss [hunk 11]
```diff
@@ -2,6 +2,16 @@ a.nav-link.waves-effect.waves-light:hover {
   color: var(--color-primary-light);
 }
 
+.navbar {
+  width: 100% !important;
+  min-width: 100% !important;
+  height: 82px !important;
+  //position: fixed !important;
```
Comment (MBenedicte) [inline src/app/modules/shared/components/navbar/navbar.component.scss]: the same as here

File: src/app/modules/shared/components/navbar/navbar.component.scss [hunk 12]
```diff
@@ -119,30 +129,57 @@ ul.top-menu > li > a:hover .caret {
 
 
 .btn-auth {
-  line-height: 18px !important;
-  letter-spacing: normal !important;
+  //line-height: 18px !important;
+  //letter-spacing: normal !important;
```
Comment (MBenedicte) [inline src/app/modules/shared/components/navbar/navbar.component.scss]: Delete these comments

File: src/app/modules/shared/components/navbar/navbar.component.scss [hunk 13]
```diff
@@ -119,30 +129,57 @@ ul.top-menu > li > a:hover .caret {
...
-  display: inline-block !important;
-  padding: 6px 12px !important;
-  position: relative !important;
-  text-align: center !important;
-  width: auto !important;
+  //display: inline-block !important;
+  //padding: 6px 12px !important;
+  //position: relative !important;
+  //text-align: center !important;
+  //width: auto !important;
```
Comment (MBenedicte) [inline src/app/modules/shared/components/navbar/navbar.component.scss]: Please delete the comments

File: src/app/services/http.service.ts [hunk 14]
```diff
@@ -12,28 +12,27 @@ export class HttpService {
...
-  httpOptions = {
-    headers: new HttpHeaders({
-       'Content-Type': 'application/json'
-    })
-  };
+  // httpOptions = {
+  //   headers: new HttpHeaders({
+  //      'Content-Type': 'application/json'
+  //   })
+  // };
```
Comment (MBenedicte) [inline src/app/services/http.service.ts:19]: Please the comments

File: src/app/app.component.ts (hunk 4 above)
Comment (kdaglas) [reply src/app/app.component.ts]: no they are not... let me remove them... thank you

File: src/app/helpers/tests/spies.ts (hunk 5 above)
Comment (kdaglas) [reply src/app/helpers/tests/spies.ts]: yes am onit... thank you

File: src/app/modules/features/components/payment/payment.module.ts (hunk 6 above)
Comment (kdaglas) [reply src/app/modules/features/components/payment/payment.module.ts]: we nolonger use it... i had made sure that we have fewer modules that will hold our components according to the structure i made... thanks for pointing this out... had not seen it

File: src/app/modules/features/components/profile/financial-information/financial-information.component.spec.ts (hunk 7 above)
Comment (kdaglas) [reply src/app/modules/features/components/profile/financial-information/financial-information.component.spec.ts:108]: this file is required... it needs refactoring so i just commented it out so that i can create a bug fix for it

File: src/app/modules/features/components/profile/financial-information/financial-information.component.ts (hunk 8 above)
Comment (kdaglas) [reply src/app/modules/features/components/profile/financial-information/financial-information.component.ts]: thanks... onit

File: src/app/modules/shared/components/navbar/navbar.component.html (hunk 10 above)
Comment (kdaglas) [reply src/app/modules/shared/components/navbar/navbar.component.html]: i have worked on this

File: src/app/app.component.spec.ts [hunk 15]
```diff
@@ -1,18 +1,44 @@
-import {async, TestBed} from '@angular/core/testing';
-import {RouterTestingModule} from '@angular/router/testing';
-import {AppComponent} from 'src/app/app.component';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from 'src/app/app.component';
+import {configureTestSuite} from 'ng-bullet';
+import {PropertiesComponent} from './components/properties/properties.component';
```
Comment (Jaman-dedy) [inline src/app/app.component.spec.ts]: Consider using absolute import in this file please

File: src/app/modules/authentication/authentication.routing.ts [hunk 16]
```diff
@@ -0,0 +1,46 @@
+import { Routes } from '@angular/router';
+import { LoginComponent } from './components/login/login.component';
```
Comment (Jaman-dedy) [inline src/app/modules/authentication/authentication.routing.ts]: Absolute import here

File: src/app/modules/shared/components/navbar/navbar.component.scss (hunk 11 above)
Comment (kdaglas) [reply src/app/modules/shared/components/navbar/navbar.component.scss]: done

File: src/app/modules/shared/components/navbar/navbar.component.scss (hunk 12 above)
Comment (kdaglas) [reply src/app/modules/shared/components/navbar/navbar.component.scss]: thanks... i have done that

File: src/app/modules/shared/components/navbar/navbar.component.scss (hunk 13 above)
Comment (kdaglas) [reply src/app/modules/shared/components/navbar/navbar.component.scss]: even this, i have worked on... thanks

File: src/app/modules/authentication/components/login/login-form/login-form.component.spec.ts [hunk 17]
```diff
@@ -7,9 +7,7 @@ import { ToastrModule, ToastrService } from 'ngx-toastr';
 import { of, throwError } from 'rxjs';
 import { LocalStorageService } from 'src/app/services/local-storage.service';
 import { LoginService } from 'src/app/services/login/login.service';
-import { localStorageSpy, loginServiceSpy, resetSpies, toastServiceSpy } from '../../../helpers/spies';
-import { LoginHeaderComponent } from '../login-header/login-header.component';
```
Comment (Jaman-dedy) [inline src/app/modules/authentication/components/login/login-form/login-form.component.spec.ts:11]: Absolute import here @kdaglas

File: src/app/modules/authentication/components/login/login.component.spec.ts [hunk 18]
```diff
@@ -6,12 +6,8 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { NgxSpinnerModule } from 'ngx-spinner';
 import { ToastrModule, ToastrService } from 'ngx-toastr';
 import { LocalStorageService } from 'src/app/services/local-storage.service';
-import { localStorageSpy, toastServiceSpy } from '../../helpers/spies';
+import { localStorageSpy, toastServiceSpy } from '../../../../helpers/tests/spies';
```
Comment (Jaman-dedy) [inline src/app/modules/authentication/components/login/login.component.spec.ts]: Absolute import here

File: src/app/services/http.service.ts (hunk 14 above)
Comment (kdaglas) [reply src/app/services/http.service.ts:19]: this snippet of code is required for the other refactor PR which will be implemented in... so will keep it here for the meantime, but will be worked on in that PR

File: src/app/services/local-storage.service.ts [hunk 19]
```diff
@@ -10,7 +10,9 @@ export class LocalStorageService {
   get(key: string, fallback: any) {
     if (isPlatformBrowser(this.platformId)) {
       const value = localStorage.getItem(key);
+      // console.log(value);
       return value ? value : fallback;
+      console.log(value);
```
Comment (Manorlds-Eaglespark) [inline src/app/services/local-storage.service.ts]: Let's remove this

File: src/app/services/local-storage.service.ts [hunk 20]
```diff
@@ -10,7 +10,9 @@ export class LocalStorageService {
   get(key: string, fallback: any) {
     if (isPlatformBrowser(this.platformId)) {
       const value = localStorage.getItem(key);
+      // console.log(value);
```
Comment (Manorlds-Eaglespark) [inline src/app/services/local-storage.service.ts]: Let's remove this

**Review by Jaman-dedy — COMMENTED** [review]
Great job @kdaglas , I tested the pr locally and it looks some functionalities are not yet working, the login and the signup button don't redirect anywhere, I can't see trending properties..

File: src/app/services/local-storage.service.ts (hunk 19 above)
Comment (kdaglas) [reply src/app/services/local-storage.service.ts]: done... thanks

File: src/app/services/local-storage.service.ts (hunk 20 above)
Comment (kdaglas) [reply src/app/services/local-storage.service.ts]: done thanks

File: src/app/app.component.spec.ts (hunk 15 above)
Comment (kdaglas) [reply src/app/app.component.spec.ts]: am working on it

File: src/app/app.routing.ts [hunk 21]
```diff
@@ -2,14 +2,21 @@ import { NgModule } from '@angular/core';
...
+import { PropertyDetailsComponent } from './components/property-details/property-details.component';
+import { NoPropertiesComponent } from './components/properties/no-properties/no-properties.component';
+import { TermsPageComponent } from './components/terms/terms.component';
+import { PropertiesComponent } from './components/properties/properties.component';
+import { RegistersuccessComponent } from './modules/authentication/components/registration/registersuccess/registersuccess.component';
+import { HomeComponent } from './components/home/home.component';
+import { AuthGuard } from './guards/auth.guard';
 
 const routes: Routes = [
   {
```
Comment (Manorlds-Eaglespark) [inline src/app/app.routing.ts:17]: Is the empty bracket necessary, kindly enlight

**Comment by Tevinthuku** [comment]
@kdaglas , what do you think of having the `input-errors folder` in the `shared/components folder` since its also a reusable component

image

File: src/app/modules/authentication/authentication.routing.ts (hunk 16 above)
Comment (kdaglas) [reply src/app/modules/authentication/authentication.routing.ts]: thank you this is done.

File: src/app/app.component.spec.ts (hunk 15 above)
Comment (kdaglas) [reply src/app/app.component.spec.ts]: done

File: src/app/app.routing.ts (hunk 21 above)
Comment (Tevinthuku) [reply src/app/app.routing.ts:17]: @Manorlds-Eaglespark , I'm not seeing empty brackets, it's just that it hasn't been modified
image

**Comment by Manorlds-Eaglespark** [comment]
Am not able to see any properties on the home page. I think something is missing here.
Screenshot 2019-09-10 at 13 47 56

**Comment by Tevinthuku** [comment]
> Am not able to see any properties on the home page. I think something is missing here.
> Screenshot 2019-09-10 at 13 47 56

I was facing the same issue, kindly pull the branch and run the server again.

**Comment by Manorlds-Eaglespark** [comment]
> > Am not able to see any properties on the home page. I think something is missing here.
> > Screenshot 2019-09-10 at 13 47 56
> 
> I was facing the same issue, kindly pull the branch and run the server again.

Done that, no changes yet. Let me try again abit later.

**Comment by Jaman-dedy** [comment]
> > Am not able to see any properties on the home page. I think something is missing here.
> > Screenshot 2019-09-10 at 13 47 56
> 
> I was facing the same issue, kindly pull the branch and run the server again.

I've pulled but still the same @Tevinthuku

File: src/app/app.routing.ts (hunk 21 above)
Comment (kdaglas) [reply src/app/app.routing.ts:17]: there is actually no empty bracket here... maybe i will sync with you on this and you tell me what is happening

File: src/app/modules/authentication/components/login/login.component.spec.ts (hunk 18 above)
Comment (kdaglas) [reply src/app/modules/authentication/components/login/login.component.spec.ts]: done

File: src/app/modules/authentication/components/login/login-form/login-form.component.spec.ts (hunk 17 above)
Comment (kdaglas) [reply src/app/modules/authentication/components/login/login-form/login-form.component.spec.ts:11]: i fixed this too, thanks

**Comment by Tevinthuku** [comment]
> > > Am not able to see any properties on the home page. I think something is missing here.
> > > Screenshot 2019-09-10 at 13 47 56
> > 
> > 
> > I was facing the same issue, kindly pull the branch and run the server again.
> 
> I've pulled but still the same @Tevinthuku

Wierd 🤔 . It works on my end.. I'm using `npm run start` I've just noticed that if you use the `ng serve -o` command it doesn't work @Jaman-dedy  @Manorlds-Eaglespark

**Comment by kdaglas** [comment]
> > > > Am not able to see any properties on the home page. I think something is missing here.
> > > > Screenshot 2019-09-10 at 13 47 56
> > > 
> > > 
> > > I was facing the same issue, kindly pull the branch and run the server again.
> > 
> > 
> > I've pulled but still the same @Tevinthuku
> 
> Wierd 🤔 . It works on my end.. I'm using `npm run start` I've just noticed that if you use the `ng serve -o` command it doesn't work @Jaman-dedy @Manorlds-Eaglespark

me too, but let me see how that issue is coming about

**Comment by kdaglas** [comment]
> @kdaglas , what do you think of having the `input-errors folder` in the `shared/components folder` since its also a reusable component
> 
> image

speaking of this, you are right, let me change that to be in components

**Comment by Niyitangasam** [comment]
> > > > > Am not able to see any properties on the home page. I think something is missing here.
> > > > > Screenshot 2019-09-10 at 13 47 56
> > > > 
> > > > 
> > > > I was facing the same issue, kindly pull the branch and run the server again.
> > > 
> > > 
> > > I've pulled but still the same @Tevinthuku
> > 
> > 
> > Wierd 🤔 . It works on my end.. I'm using `npm run start` I've just noticed that if you use the `ng serve -o` command it doesn't work @Jaman-dedy @Manorlds-Eaglespark
> 
> me too, but let me see how that issue is coming about

I think that we have to use `npm run start` to start it because as I remember  @CryceTruly changed the way our app starts in the other PR of metadata and `ng serve -o` is no longer working.

**Comment by JamesMudidi** [comment]
> Hey @kdaglas , nice work, Im getting this error though, right in the home page.
> image

This issue is puzzling since it comes in when tested locally but it's passing on Heroku as seen
[here](https://landville-frontend-pr-58.herokuapp.com/home)

**Comment by kdaglas** [comment]
> > Hey @kdaglas , nice work, Im getting this error though, right in the home page.
> > image
> 
> This issue is puzzling since it comes in when tested locally but it's passing on Heroku

so according to @CryceTruly z PR, our `start` command was changed and the `ng serve -o` is no longer functional... so try to use `npm start` or npm run start`... but in the meantime am looking into a better way to have all the commands working

**Comment by kdaglas** [comment]
> @kdaglas this is a good job, I would like to point some few concerns:
> 
> **1. I have noticed that there is a change in the navbar for the `login` and `signup` buttons.**
> 
> Screen Shot 2019-09-09 at 20 01 19
> 
> **I think these buttons would look better with the existing design that was there**
> 
> Screen Shot 2019-09-09 at 20 03 01
> 
> **2. Also, the design for the dropdown menu on small screens has changed**
> 
> Screen Shot 2019-09-09 at 20 45 30
> 
> **The design that was there**
> 
> Screen Shot 2019-09-09 at 20 46 14
> 
> **3. I have noticed that after logging in, the `navbar` doesn't change accordingly, it waits until you refresh. means I can still see the login and sign up buttons when logged in.**
> 
> Screen Shot 2019-09-09 at 20 15 08

I have fixed this, I was testing out some functionality and UI implementation but I have fixed this

**Comment by Tevinthuku** [comment]
@kdaglas , I've noticed, once a user logs into the app and redirected back to the home-page the navigation doesn't update accordingly 
image

unlike the previous version

image

Will this be 1 of the bugs that will need fixing?

File: src/app/components/properties/properties.component.spec.ts [hunk 22]
```diff
@@ -33,14 +32,14 @@ describe('PropertiesComponent', () => {
...
+  configureTestSuite(() => {
     TestBed.configureTestingModule({
-      declarations: [PropertiesComponent, CardComponent],
+      declarations: [],
       imports: [
         AppModule,
         HttpClientModule,
         NgxSpinnerModule,
-        AuthLayoutModule,
+        // AuthenticationModule,
```
Comment (Rhytah) [inline src/app/components/properties/properties.component.spec.ts]: @kdaglas there's another comment that seems to have evaded your attention.

File: src/app/interceptors/jwt/jwt.interceptor.spec.ts [hunk 23]
```diff
@@ -1,6 +1,6 @@
 import { TestBed } from '@angular/core/testing';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { httpHandlerSpy, httpRequestSpy } from './spies';
+import { httpHandlerSpy, httpRequestSpy } from '../../helpers/tests/spies';
```
Comment (Rhytah) [inline src/app/interceptors/jwt/jwt.interceptor.spec.ts]: @kdaglas we can transform this into an absolute path

File: src/app/interceptors/error/error.interceptor.spec.ts [hunk 24]
```diff
@@ -1,7 +1,7 @@
 import {TestBed} from '@angular/core/testing';
 import {HttpClientTestingModule} from '@angular/common/http/testing';
 import {ErrorInterceptor} from './error.interceptor';
-import {httpHandlerSpy, httpRequestSpy, localStorageSpy, routerSpy} from './spies';
+import {httpHandlerSpy, httpRequestSpy, localStorageSpy, routerSpy} from '../../helpers/tests/spies';
```
Comment (Rhytah) [inline src/app/interceptors/error/error.interceptor.spec.ts]: @kdaglas change this path to absolute as well

File: src/app/interceptors/jwt/jwt.interceptor.spec.ts (hunk 23 above)
Comment (kdaglas) [reply src/app/interceptors/jwt/jwt.interceptor.spec.ts]: on it... thanks

File: src/app/components/properties/properties.component.spec.ts (hunk 22 above)
Comment (kdaglas) [reply src/app/components/properties/properties.component.spec.ts]: on it... thanks

**Review by dorothyas — APPROVED** [review]
LGTM

**Review by Jaman-dedy — APPROVED** [review]

**Review by JamesMudidi — APPROVED** [review]
LGTM

**Comment by kdaglas** [comment]
> @kdaglas , I've noticed, once a user logs into the app and redirected back to the home-page the navigation doesn't update accordingly
> image
> 
> unlike the previous version
> 
> image
> 
> Will this be 1 of the bugs that will need fixing?

yes this is going to be a bug fix

File: src/app/interceptors/error/error.interceptor.spec.ts (hunk 24 above)
Comment (kdaglas) [reply src/app/interceptors/error/error.interceptor.spec.ts]: done

**Review by Tevinthuku — APPROVED** [review]

**Review by patrickf949 — APPROVED** [review]

**Review by neelxie — APPROVED** [review]

File: src/app/app.component.scss [hunk 25]
```diff
@@ -18,4 +18,4 @@ hr.header-line {
   hr.header-line {
     width: 135%;
   }
-}
\ No newline at end of file
+}
```
Comment (MBenedicte) [inline src/app/app.component.scss]: Add the empty line

File: src/styles.scss [hunk 26]
```diff
@@ -4,3 +4,33 @@
 @import '~ngx-toastr/toastr-bs4-alert';
 @import '~ngx-toastr/toastr';
 @import '../node_modules/ngx-toastr/toastr.css';
+
+//body {
+//  margin: 0 auto;
+//  width: 75%;
+//}
```
Comment (MBenedicte) [inline src/styles.scss]: Please remove the comments

File: src/styles.scss [hunk 27]
```diff
@@ -4,3 +4,33 @@
...
+  font-family: 'Airbnb Cereal App Bold';
+  font-style: normal;
+  font-weight: normal;
+  src: local('./assets/fonts/cereal-font/Airbnb Cereal App Bold'), url('./assets/fonts/cereal-font/AirbnbCerealBold.woff') format('woff');
+}
+
+p {
+  font-family: 'Airbnb Cereal App Light', Arial, sans-serif;
+  //font-weight:normal;
+  //font-style:normal;
```
Comment (MBenedicte) [inline src/styles.scss]: Remove the comments

**Comment by shemaeric** [comment]
what do you think of putting these file in respective folders, like other services. For auth, local-storage and http services.
Screen Shot 2019-09-10 at 16 18 56

**Review by MBenedicte — APPROVED** [review]

**Comment by kdaglas** [comment]
> what do you think of putting these file in respective folders, like other services. For auth, local-storage and http services.
> Screen Shot 2019-09-10 at 16 18 56

this is being worked in another PR that is refactoring the services @shemaeric

**Review by alexxsanya — COMMENTED** [review]
Screenshot 2019-09-10 at 17 50 23

@kdaglas well done on this though I have just noticed that the highlighted menu items have a different font size of 0.9rem from the rest which are 1.0rem. I suggest you fix this.

**Comment by kdaglas** [comment]
> Screenshot 2019-09-10 at 17 50 23
> 
> @kdaglas I just noticed that the highlighted menu items have a different font size of 0.9rem from the rest which are 1.0rem. I suggest you fix this.

This is going to be fixed in another PR the one for the navbar component and the login component

**Review by shemaeric — APPROVED** [review]

**Review by Manorlds-Eaglespark — APPROVED** [review]

**Review by alexxsanya — APPROVED** [review]
LGTM. @kdaglas well done on this.

**Review by CryceTruly — APPROVED** [review]

**Review by Niyitangasam — APPROVED** [review]
LGTM

**Review by Victoradukwu — APPROVED** [review]

**Review by Rhytah — APPROVED** [review]

**Review by espoirMur — APPROVED** [review]

**Review by bl4ck4ndbr0wn — APPROVED** [review]
Great work. @kdaglas, love the new folder structure and the separation of concerns. 
LGTM

**Review by Tevinthuku — APPROVED** [review]

**Comment by malep2007** [comment]
> Screenshot 2019-09-10 at 17 50 23
> 
> @kdaglas well done on this though I have just noticed that the highlighted menu items have a different font size of 0.9rem from the rest which are 1.0rem. I suggest you fix this.

@alexxsanya add an issue so we can keep track of this.
